README.md

```diff
@@ -16,32 +16,51 @@ environment:
-| `readinessPeriod`          | Readiness probe period (seconds)                          | `15`                                                                              |
-| `livenessPath`             | Path of HTTP liveness probe                               | `/health`                                                                         |
-| `livenessDelay`            | Liveness probe inital delay (seconds)                     | `30`                                                                              |
-| `livenessTimeout`          | Liveness probe timeout (seconds)                          | `3`                                                                               |
-| `livenessPeriod`           | Liveness probe period (seconds)                           | `15`                                                                              |
-| `livenessFailureThreshold` | Liveness failure threshold                                | `3`                                                                               |
+| Parameter                  | Description                                | Default  |
+| -------------------------- | ------------------------------------------ | ----- |
+| `applicationPort`          | The port your app runs on in its container | `4550`|
+| `image`                    | Full image url | `hmctssandbox.azurecr.io/hmcts/spring-boot-template`<br>(but overridden by pipeline) |
+| `environment`              |  A map containing all environment values you wish to set. <br> **Note**: environment variables (the key in KEY: value) must be uppercase and only contain letters,  "_", or numbers | `nil`|
+| `configmap`                | A config map, can be used for environment specific config.| `nil`|
+| `memoryRequests`           | Requests for memory | `512Mi`|
+| `cpuRequests`              | Requests for cpu | `100m`|
+| `memoryLimits`             | Memory limits| `1024Mi`|
+| `cpuLimits`                | CPU limits | `2500m`|
+| `ingressHost`              | Host for ingress controller to map the container to | `chart-java.service.core-compute-saat.internal`<br>(but overridden by pipeline)|
+| `readinessPath`            | Path of HTTP readiness probe | `/health`|
+| `readinessDelay`           | Readiness probe inital delay (seconds)| `30`|
+| `readinessTimeout`         | Readiness probe timeout (seconds)| `3`|
+| `readinessPeriod`          | Readiness probe period (seconds) | `15`|
+| `livenessPath`             | Path of HTTP liveness probe | `/health`|
+| `livenessDelay`            | Liveness probe inital delay (seconds)  | `30`|
+| `livenessTimeout`          | Liveness probe timeout (seconds) | `3`|
+| `livenessPeriod`           | Liveness probe period (seconds) | `15`|
+| `livenessFailureThreshold` | Liveness failure threshold | `3`                                                     |
+| `secrets`                  | list of all the secret variables to get from an attached secret volume. |  nil |
```

Comment:
Might want to change this to say something like:
```
mappings of environment variables to pre-configured kubernetes secrets objects
```

Reply:
sure ... my terminology is not the best atm.